Sources/FluentSQL/SQLQueryConverter.swift

```diff
@@ -172,6 +173,13 @@ public struct SQLQueryConverter {
             default:
                 fatalError("Deep SQL JSON nesting not yet supported.")
             }
+        case .function(let function, let fields):
```

Comment:
I think we could achieve this without `DatabaseQuery.Function` by checking `query.isDistinct` here.

Reply:
There is a difference. Becuase `SELECT DISTINCT userId FROM users` is not equal to, say, `SELECT COUNT(DISTINCT(userId)) FROM users`. How would you otherwise differentiate between just prefixing `DISTINCT` and using it inside an aggregation like `COUNT`